docs/pages/fragments/high-availability.mdx

```diff
@@ -57,8 +57,57 @@ vcluster connect vcluster-1 -n host-namespace-1
 
 # Then execute in a new terminal
 export KUBECONFIG=kubeconfig.yaml
-kubectl get ns
+kubectl get ns  
 ...
 ```
 
+## Enable HA in rootless mode
+
+```
+# Enable HA mode
+enableHA: true
+
+# Scale up syncer replicas
+syncer:
+  replicas: 3
+  securityContext:
+    runAsGroup: 12345
+    runAsNonRoot: true
+    runAsUser: 12345
+
+# Scale up etcd
+etcd:
+  replicas: 3
+  fsGroup: 12345
+  securityContext:
+    runAsGroup: 12345
+    runAsNonRoot: true
+    runAsUser: 12345
+
+# Scale up controller manager
+controller:
+  replicas: 3
+  securityContext:
+    runAsGroup: 12345
+    runAsNonRoot: true
+    runAsUser: 12345
+
+# Scale up api server
+api:
+  replicas: 3
+  securityContext:
+    runAsGroup: 12345
+    runAsNonRoot: true
+    runAsUser: 12345
+
+# Scale up DNS server
+coredns:
+  replicas: 3
+  securityContext:
+    runAsGroup: 12345
+    runAsNonRoot: true
+    runAsUser: 12345
+```
+You can also refer to [this example](https://github.com/loft-sh/vcluster/issues/674) for HA in rootless mode.
+
 Check the [github repository](https://github.com/loft-sh/vcluster/tree/main/charts/k8s) for all available chart options.
```

Comment:
The issue has a pretty descriptive example so I thought it could be helpful if someone wants something exactly like that. Here in docs, we can just give a minimalistic config for HA in rootless mode.